incubator/chartmuseum/README.md

```diff
@@ -1,5 +1,78 @@
 # ChartMuseum Helm Chart
 
-Work in progress...
+Deploy your own private ChartMuseum.   
 
-Please see https://github.com/chartmuseum/chartmuseum
\ No newline at end of file
+Please also see https://github.com/chartmuseum/chartmuseum   
+
+## Prerequisites
+
+* Kubernetes with extensions/v1beta1 available
+* [If enabled] A persistent storage resource and RW access to it
+* [If enabled] Kubernetes StorageClass for dynamic provisioning
+
+## Configuration
```

Comment:
Can we add a comment here to say "check out values.yaml for all the possible configuration options" ?